app/controllers/admin/api/cms/templates_controller.rb

```diff
@@ -48,7 +55,7 @@ def index
   ##~ op.parameters.add :name => "liquid_enabled", :description => "liquid processing of the template content on/off", :paramType => "query", :type => "boolean"
   ##~ op.parameters.add :name => "handler", :paramType => "query", :description => "text will be processed by the handler before rendering", :required => false, :allowableValues => { :valueType => "LIST", :values => ["textile", "markdown"]  }
   def create
-    template = Admin::Api::CMS::TemplateService::Create.call(current_account, cms_template_params)
+    template = Admin::Api::CMS::TemplateService::Create.call(current_account, params, cms_template_params)
```

Comment:
This change comes from https://github.com/3scale/porta/pull/3228/commits/5eba7a04088e05a640bdb90bfcd148f93ff68367. The service was updated to prevent writing the  `type` column.